templates/commands/render/render.go

```diff
@@ -495,10 +510,40 @@ func checkInputsMissing(spec *spec.Spec, inputs map[string]string) []string {
 func executeSteps(ctx context.Context, steps []*spec.Step, sp *stepParams) error {
 	logger := logging.FromContext(ctx).With("logger", "executeSteps")
 
+	if sp.flags.DebugStepDiffs {
+		argsList := [][]string{
+			// Make debug dir a git repository with a detached work tree in scratch
+			// dir, meaning it will track the file changes in scratch dir without
+			// affecting the scratch dir.
+			strings.Fields(fmt.Sprintf(
```

Comment:
Rather than use strings.Fields, I think it would be a little cleaner to just declare the args as a string slice, like

```
{"--git-dir", sp.debugDir, "--work-tree", sp.scratchDir}
```

This avoids some possible bugs if debugDir or scratchDir somehow contain whitespace.

Reply:
Make sense, thanks!